lib/renderer/ipc-renderer-internal.ts

```diff
@@ -1,10 +1,8 @@
-'use strict'
-
 const binding = process.atomBinding('ipc')
 const v8Util = process.atomBinding('v8_util')
 
 // Created by init.js.
-const ipcRenderer = v8Util.getHiddenValue(global, 'ipc-internal')
+export const ipcRenderer: Electron.IpcRenderer = v8Util.getHiddenValue(global, 'ipc-internal')
```

Comment:
Can we copy the main process here and `export const ipcRendererInternal: ...` to enforce a naming convention in our code 👍 (So we don't get confused between `ipcRenderer` and `ipcRendererInternal` 👍 )

Reply:
Good callout! I've updated the whole PR.